### PR TITLE
1.99.2 relnotes STO-7735

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Preview links

- [Version 1.99.2](https://6674c810076d6eeecc66687f--harness-developer.netlify.app/release-notes/security-testing-orchestration#version-1992)
- Version 1.96.2 > [Breaking change](https://6674c810076d6eeecc66687f--harness-developer.netlify.app/release-notes/security-testing-orchestration#version-1992)

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/security-testing-orchestration.md [hunk 1]
```diff
@@ -97,7 +73,37 @@ The following features are now generally available:
...
+  1) Added another API call to resolve a Burp schedule item iD to its corresponding latest Burp scan ID.
+  2) Added logic to perform updates on matching Burp sites rather than trying to create a new Burp site with the same name.
+  3) Removed default port from being set by runner and will only set port if user specifies in the step.
+
+
+### Version 1.97
+
+#### New feature
+
+- The STO API is now generally available and publicly documented. For more information, go to [STO](https://apidocs.harness.io/tag/Exemptions) in the Harness API documentation. (STO-5281)
```
Comment (ahokinson) [inline release-notes/security-testing-orchestration.md:104]: Linking directly to Exemptions is odd. Would Scans/Issues be more appropriate? I see we can't link to an STO section of the page, but I wonder if we should change this?

File: release-notes/security-testing-orchestration.md [hunk 2]
```diff
@@ -116,6 +122,12 @@ The following features are now generally available:
 
 <!-- 2024-05-22 -->
 
+###### Breaking change
+
+Harness introduced a back-end dashboard change to support features that are currently in development. As a result, you must re-create any custom dashboards created before this change was introduced. If the old dashboard had scheduling enabled, you must also re-create the schedule. (STO-7199)
```
Comment (ahokinson) [inline release-notes/security-testing-orchestration.md:127]: Is it true that *any* custom dashboards must be re-created? I thought it was only specific dashboards. I could be wrong though.

File: release-notes/security-testing-orchestration.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:104]: @ahokinson I couldn't figure out a way to link to the STO heading so I just linked to the first subheading under it.

**Comment by douglas-j-bothwell** [comment]
Closing this PR due to the doc reorganization. Revisiting here: https://github.com/harness/developer-hub/pull/7179

File: release-notes/security-testing-orchestration.md (hunk 2 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:127]: @GabeAboy @ahokinson 

Revised as follows:

Harness introduced a back-end dashboard change to support features that are currently in development. As a result, you must re-create any custom dashboards cloned from the Security Testing Dashboard before this change was introduced. If the cloned dashboard has scheduling enabled, you must also re-create the schedule. (STO-7199)

https://6679fa4233e46c5191f1fd40--harness-developer.netlify.app/release-notes/security-testing-orchestration#breaking-change

**Pull request closed without merging**